app/views/compute_resources_vms/show/_azurerm.html.erb

```diff
@@ -1,11 +1,12 @@
-<% title @vm.name %>
+<% title @vm.azure_vm.name %>
 <div class='col-md-12'>
   <table class="<%= table_css_classes %>">
     <thead>
     <tr><th colspan="2"><%=_('Properties') %></th></tr>
     </thead>
     <tbody>
       <%= prop :id %>
+      <!--values accessible through @vm.azure_vm.location/platform-->
       <%= prop :location %>
```

Comment:
How should we set the property values here for location and platform here since they are wrapped inside azure_vm?